### PR TITLE
#41 re-designed login page

### PR DESCRIPTION
**Pull request by 0xYuval**
image

### CODE REVIEW

File: src/app/users/user-login/user-login.component.html [hunk 1]
```diff
@@ -1,8 +1,24 @@
-<button class="facebook" (click)="signInWithFacebook()">
-  <fa name="facebook"></fa>
-  Continue with Facebook
-</button>
-<button class="guest" routerLink="/">
-  <fa name="user"></fa>
-  Continue as guest
-</button>
+<div class="col-sm-6 center" id="inner" style="width:35%">
```
[the remark this answers is not in the document]
Comment (menhel) [reply src/app/users/user-login/user-login.component.html]: remove the `id="inner"`. It is logically incorrect.
id should be the unique identifier of this element in this component. the name you chose, inner, is not specific enough nor do you really need to assign an id to this element.

[the remark this answers is not in the document]
Comment (menhel) [reply src/app/users/user-login/user-login.component.html]: In Bootstrap, cols are always contained in a row. That's the way a bootstrap grid works.
wrap this entire div in div with class `row`.

[the remark this answers is not in the document]
Comment (menhel) [reply src/app/users/user-login/user-login.component.html]: In most cases in bootstrap, rows are contained in a container.
wrap the row div you just added in a div with class `container`

File: src/app/users/user-login/user-login.component.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#inner {
```
Comment (menhel) [inline src/app/users/user-login/user-login.component.scss]: As all the design issues will be solved using default bootstrap classes, there is no need for content inside the css at all. Remove all content.

File: src/app/users/user-login/user-login.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (menhel) [reply src/app/users/user-login/user-login.component.html]: To center the col inside it's row element, use the `align-self-center`  class to the col div.
There is a slight chance I don't remember the exact class name, or exactly how to use it. If this does not center the col inside the row, check the bootstrap documentation for the correct usage.

[the remark this answers is not in the document]
Comment (yuvalron) [reply src/app/users/user-login/user-login.component.html]: Got it. By the way, in order to center the col inside a row i used `justify-content-center` row class.

**Pull request merged**